examples/init_speedy.js

```diff
@@ -56,15 +58,20 @@ async function loadSpeedyVideo(id) {
 
         ctx.drawImage(image.source, 0, 0);
 
-        let dt = ctx.getImageData(0, 0, oWidth, oHeight);
+        data = ctx.getImageData(0, 0, oWidth, oHeight);
+
+        //console.log(data);
```

Comment:
if you comment out this console.log it will ever return an ImageData with a correct (not empty) Uint8ClampedArray